.github/workflows/build_and_release.yml

```diff
@@ -91,9 +91,8 @@ jobs:
     # Use them with `-j 4` flag
     - name: Build Library
       run: |
-        #python setup.py bdist_wheel -j 4
-        #auditwheel repair --plat manylinux_2_31_x86_64 dist/*.whl
-        pip install .
+        python setup.py bdist_wheel -j 4
+        auditwheel repair --plat manylinux_2_31_x86_64 dist/*.whl
```

Comment:
Can you add comment why did you pick `manylinux_2_31_x86_64` (perhaps link on original from where this was taken)

Reply:
Sorry I can't. 
Initial commit change this ` .github/workflow/build_and_release.yml` file then I revert it, in the latest commit. So I guess you are the original author of this setting.